# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: architecture/arm64/package/*/apple-t600x-sdhci.patch
# Copyright (C) 2021 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- linux/arch/arm64/boot/dts/apple/t600x-j314-j316.dtsi.vanilla	2021-11-25 16:34:52.410908567 +0100
+++ linux/arch/arm64/boot/dts/apple/t600x-j314-j316.dtsi	2021-11-25 16:35:36.172910824 +0100
@@ -145,7 +145,7 @@
 	/* SD card reader */
 	bus-range = <2 2>;
 	max-link-speed = <1>;
-	sdhci0: ethernet@0,0 {
+	sdhci0: pci@0,0 {
 		reg = <0x20000 0x0 0x0 0x0 0x0>;
 		cd-inverted;
 		wp-inverted;
